tests/test_alerting_works.py

```diff
@@ -0,0 +1,18 @@
+import logging
```

Comment:
The file should be under tests/UI ?

Reply:
Moved, although most of alerting tests will be probably under `tests/manage` directory.